frontend/routes/Wifi.jsx

```diff
@@ -12,42 +29,235 @@ export class Component extends React.Component {
       wifi_networks: [],
       last_refresh: '',
     };
+
     setInterval(() => {
-      exec_table('nmcli dev wifi').then(result => this.setState({ wifi_networks: result }));
-      exec('date').then(result => this.setState({ last_refresh: result }));
+      exec_table('nmcli dev wifi').then(result => {
+        if (!result) return;
+
+        this.setState({
+          wifi_networks: result,
+        });
+      });
+      exec('date').then(result => {
+        this.setState({
+          last_refresh: result,
+        });
+      });
     }, 1000);
   }
 
   render() {
     return (
       <div>
-        <Table data={this.state.wifi_networks} />
+        <EnhancedTable rows={this.state.wifi_networks} />
       </div>
     );
   }
 }
 
-function Table(props) {
-  const { data } = props;
-  if (data.length === 0) return <table />;
+
+// Meta-Data
+
+const in_use = 'IN-USE';
+const first_column = 'IN-USE';
+const active_color = orange[50];
+const headCells = {
+  'IN-USE': { numeric: false, disablePadding: true, label: 'Connected' },
```

Comment:
See above.

Reply:
Hey, I had a doubt in this. How can I replace IN-USE here, there is a hyphen. How can i make this consistent? I have tried making other keys with quotes but `npm run format` removes the quotes except for `IN-USE`

Reply:
Should I remove the constant and use 'IN-USE' everywhere ?

Reply:
I think having the constant is fine, but if you have a constant please use it everywhere,  (You can use it in the dictionary like `const headCells = { [in_use] : .... }`

Reply:
Done